src/styles/_globals.scss

```diff
@@ -265,22 +260,17 @@ span[data-pf2-effect-area] {
 
 i[data-pf2-repost] {
```

Comment:
The new repost can be seen in the image in the description

Reply:
Removed this stuff from this PR and put it in https://github.com/foundryvtt/pf2e/pull/5132